### DISCUSSION TITLE
请问一下这个筛选出来的ip对应的端口号信息在哪查看啊？

### DISCUSSION THREAD

**Discussion by spotiex** [XIU2/CloudflareSpeedTest#397] [讨论]
如题，有无一种 [ ip : port ]的筛选结果查看方法，或者说本人没有找到这个功能，谢谢大家

**Comment by XIU2** [comment]
软件测速端口是通过 `-tp 443` 参数指定的，默认是 443 。
每次测速都只是测试固定的一个端口，因此不存在什么 IP:端口 筛选结果方式什么的。

> 因为 CloudflareST 是给网站自选 IP 用的，而网站的 IP 都是固定的这几个（80 443 8080 等），而且 CDN 也限制端口。

**Comment by spotiex** [comment]
XIU2 ***@***.***>於 2023年6月24日 週六，21:01寫道：

> 软件测速端口是根据 -tp xxx 参数指定的，默认是 443.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/XIU2/CloudflareSpeedTest/discussions/397#discussioncomment-6269106>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A57XZX6T5VOPJ2MYO6CSC7LXM3QLPANCNFSM6AAAAAAZSPUS3I>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
感谢感谢，但是一直占用443有时候有冲突啊...

**Reply by XIU2** [reply]
啥玩意？占用 443？
CloudflareST 是访问 IP:443 测速的（发），而不是监听 443 端口（收），何来占用 443 端口？

**Comment by spotiex** [comment]
也可能是我操作不当，网上看的其他优选教程是筛选的ip:port这样的形式，然后直接导入tun文件，所以我来问一下有没有这种

Cin Piz ***@***.***>於 2023年6月24日 週六，23:58寫道：

**Reply by XIU2** [reply]
我都不知道你在说什么。。。
我感觉我们两个讨论这么多，似乎不在一个频道上，似乎说的都不是一个东西。

我只回答和 CloudflareST 本身有关的问题，对于其他如代理什么的我就不管了。

**Comment by spotiex** [comment]
XIU2 ***@***.***>於 2023年6月25日 週日，12:43寫道：

> 我都不知道你在说什么。。。
> 我感觉我们两个讨论这么多，似乎不在一个频道上，似乎说的都不是一个东西。
>
> 我只回答和 CloudflareST 本身有关的问题，对于其他如代理什么的我就不管了。
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/XIU2/CloudflareSpeedTest/discussions/397#discussioncomment-6272122>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A57XZXYJKP3746VOPFD7TSTXM66VVANCNFSM6AAAAAAZSPUS3I>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>
OK，感谢你们的付出，祝你们越做越好

**Comment by jiucaiguo** [comment]
是的。我也发现这问题。在macos里运行得到的优选ip没有显示对应的端口号。导致无法使用。

**Reply by XIU2** [reply]
项目介绍里写了：

> 注意！本软件仅适用于网站，不支持给 Cloudflare WARP 优选 IP，具体见：#392
